org.eclipse.m2e.core/src/org/eclipse/m2e/core/internal/messages.properties

```diff
@@ -110,7 +108,7 @@ ProjectConfigurationManager_task_importing2=Importing project {0}
 ProjectConfigurationManager_task_refreshing=Refreshing projects
 ProjectConfigurationManager_task_updating=Updating configuration for {0}
 ProjectConfigurationManager_task_updating_projects=Updating Maven projects
-ProjectRegistryManager_task_project=project {0}
+ProjectRegistryManager_task_project=Project ''{0}''
```

Comment:
Are the double single quotes necessary (Sorry I'm only on my phone and can't try it out atm.)?